zerver/lib/bugdown/__init__.py

```diff
@@ -865,7 +877,10 @@ def run(self, root: Element) -> None:
             youtube = self.youtube_image(url)
             if youtube is not None:
                 yt_id = self.youtube_id(url)
-                add_a(root, youtube, url, None, None, "youtube-video message_inline_image", yt_id)
+                yt_title = "YouTube - {}".format(self.get_youtube_title(url))
```

Comment:
Check whether get_youtube_title is returning None. if yes then execute `continue` statement. 
Else do `yt_title = "YouTube - {}".format(self.get_youtube_title(url))`